### PR TITLE
add personetics categories and subcategories + update m2m security co…

### PR DESCRIPTION
**Pull request by leo-dcfa**
…mponent

### CODE REVIEW

File: data-banking.yaml [hunk 1]
```diff
@@ -327,14 +337,169 @@ components:
...
+            - "Travel"
+            - "Utilities"
+            - "Other Deposits"
+            - "Other transfers"
+            - "Other"
         subCategoryName:
           type: string
+          example: "Cash Deposit"
+          enum:
+            - "Cash Deposit"
```
Comment (arthuradr) [inline data-banking.yaml:399]: can we please have sub category in alphabetic order as well?

File: data-banking.yaml [hunk 2]
```diff
@@ -327,14 +337,169 @@ components:
...
+            - "Transfers Between Own Accounts"
+            - "Transportation"
+            - "Travel"
+            - "Utilities"
+            - "Other Deposits"
+            - "Other transfers"
+            - "Other"
         subCategoryName:
           type: string
+          example: "Cash Deposit"
```
Comment (arthuradr) [inline data-banking.yaml:383]: maybe not for now, can we help our customer to map a sub category name into a category?

Comment (leo-dcfa) [reply data-banking.yaml:383]: I will follow up with Personetics on this one

File: data-banking.yaml (hunk 1 above)
Comment (leo-dcfa) [reply data-banking.yaml:399]: Done

**Review by arthuradr — APPROVED** [review]

**Pull request merged**